src/main/java/net/snowflake/client/core/bind/BindUploader.java

```diff
@@ -64,6 +68,9 @@
   // size (bytes) per file in upload, 100MB default
   private long fileSize = 100 * 1024 * 1024;
 
+  // size (bytes) of max input stream
+  private long inputStreamBufferSize = 12 * 1024;
```

Comment:
Size of input stream buffer. I did speed tests with the size, and the larger the size, the faster it seems to go. However, I don't know how big is too big. Tests results are here: https://docs.google.com/spreadsheets/d/15Ox-8TS8ZZJ90fBZLYa9q21iDGgVcVP6wy31yc_ATA8/edit?usp=sharing